GUIDE.md

```diff
@@ -45,11 +45,11 @@ You can run es-node from a pre-built Docker image, a pre-built executable, or fr
 
 ## From pre-built executables
 
-Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs) and [snarkjs](#install-snarkjs).
+Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs), [snarkjs](#install-snarkjs) and [WSL](https://learn.microsoft.com/en-us/windows/wsl/) for Window. 
 
 Download the pre-built package suitable for your platform:
 
-Linux x86-64 or AMD64:
+Linux x86-64 or AMD64 or Windows:
```

Comment:
I don't think we have pre-built package for Windows, you may want to mention that for Windows users, you can run Linux version on WSL